tests/integration_tests/test_hyperopt_ray.py

```diff
@@ -123,6 +129,37 @@ def ray_cluster_4cpu():
         yield
 
 
+class HyperoptTestCallback(tune.Callback):
+    def __init__(self):
+        self.trial_ids = set()
+        self.user_config = {}
+        self.rendered_config = {}
+
+    def on_trial_start(self, iteration: int, trials: List["Trial"], trial: "Trial", **info):
+        super().on_trial_start(iteration, trials, trial, **info)
+        self.trial_ids.add(trial.trial_id)
+
+    def on_trial_complete(self, iteration: int, trials: List["Trial"], trial: "Trial", **info):
+        super().on_trial_start(iteration, trials, trial, **info)
```

Comment:
I think this should be 

```
super().on_trial_complete(iteration, trials, trial, **info)
```

Reply:
Fixed in [a073cc0](https://github.com/ludwig-ai/ludwig/pull/2640/commits/a073cc00aec3e3915c4cdd143aabbee7a247c816)